src/loaders/compute_tools/eggnog/eggnog.py

```diff
@@ -0,0 +1,58 @@
+"""
+Run eggNOG tool on a set of faa files.
+
+This tool serves a distinct purpose separate from collection tools; instead, it is suited for CDM work.
+Therefore, the parser program is not compatible with data generated by this tool.
+
+"""
+import json
+from pathlib import Path
+
+from src.loaders.common.loader_common_names import EGGNOG_METADATA
+from src.loaders.compute_tools.tool_common import ToolRunner, run_command
+
+INPUT_TYPE = 'proteins'
+THREADS = 16
+
+
+def _run_eggnog_single(
+        tool_safe_data_id: str,
+        data_id: str,
+        source_file: Path,
+        output_dir: Path,
+        debug: bool) -> None:
+
+    metadata_file = output_dir / EGGNOG_METADATA
+    if metadata_file.exists():
+        print(f"Skipping {source_file} as it has already been processed.")
+        return
+
+    # RUN eggNOG for a single genome
+    command = ['emapper.py',
+               '-i', source_file,  # Input file.
+               '-o', output_dir / source_file.name,  # Output prefix.
+                                                     # Save result file to collectiondata directory. Expecting 'emapper.annotations', 'emapper.hits' and  'emapper.seed_orthologs' files.
+               '--itype', f'{INPUT_TYPE}',
+               '--cpu', f'{THREADS}',
+               '--excel',
+               '--sensmode', 'fast',
+               '--dmnd_iterate', 'no',
```

Comment:
We don't have any mechanism to submit custom arguments to these tools do we?

Reply:
yea. We didn't build that. Something should consider if we were going to rewrite this for CDM.

Reply:
I can def see us wanting to be able to change parameter sets

Reply:
Anyway :+1: